configs/eslint-config-supensive-common/.eslintrc.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: true,
+  },
+  globals: { JSX: true },
+  parser: '@typescript-eslint/parser',
+  extends: ['airbnb', 'turbo', 'plugin:import/recommended', 'plugin:import/typescript', 'prettier'],
+  plugins: ['@typescript-eslint', 'import', 'prettier'],
+  overrides: [
+    {
+      files: ['*.spec.ts*', '*.test.ts*'],
+      plugins: ['jest'],
+      extends: ['plugin:jest/recommended'],
+      rules: { 'jest/prefer-expect-assertions': 'off' },
+    },
+    {
+      files: ['*.test-d.ts*', '*.test-d.ts*'],
+      rules: { 'import/no-unresolved': ['error', { ignore: ['tsd'] }] },
+    },
+  ],
+  rules: {
+    'consistent-return': ['off'],
+    'no-shadow': ['off'],
+    'no-use-before-define': ['off'],
+    'react/destructuring-assignment': ['off'],
+    'react/no-unescaped-entities': ['off'],
+    'react/state-in-constructor': ['off'],
+    'react/prop-types': ['off'],
+    'react/require-default-props': ['off'],
+    'react/jsx-no-useless-fragment': ['off'],
+    'react/jsx-props-no-spreading': ['off'],
+    'react/no-unstable-nested-components': ['off'],
+    'react/function-component-definition': ['off'],
+    'react/no-unused-prop-types': ['off'],
+    'react/react-in-jsx-scope': ['off'],
+    'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx', '.ts', '.tsx'] }],
```

Comment:
There are a lot of things related to react in the eslint-config-suspensive-common
I wonder what the role of `eslint-config-suspensive-react` is and what is expected to be added in the future

Reply:
I merged eslint-config-suspensive-common/react/next to `eslint-config-suspensive`

We can use like below
```js
// .eslintrc.js
module.exports = {
  root: true,
  extends: ['suspensive/react'], // we can use `suspensive/react` as subpath eslint-config-suspensive
  parserOptions: {
    tsconfigRootDir: __dirname,
    project: './tsconfig.json',
  },
}

```

I didn't know you could use `/` like `suspensive/react`, but I could. I got the idea when I saw that eslint-config-airbnb-typescript was using it like `airbnb-typescript/base`.

#### reference: [eslint-config-airbnb-typescript/base](https://www.npmjs.com/package/eslint-config-airbnb-typescript)
```js
// .eslintrc.js
module.exports = {
  root: true,
  extends: ['airbnb-base', 'airbnb-typescript/base']
}

```